constructor/header.sh

```diff
@@ -423,6 +423,14 @@ export TMP_BACKUP="${TMP:-}"
 export TMP="$PREFIX/install_tmp"
 mkdir -p "$TMP"
 
+# Check whether the virtual specs can be satisfied
+# shellcheck disable=SC2050
+if [ "__VIRTUAL_SPECS__" != "" ]; then
+    CONDA_QUIET="$BATCH" \
+    CONDA_SOLVER="classic" \
```

Comment:
Why force the classic solver here instead of using the default?

Reply:
Due to this bug: https://github.com/conda/conda-libmamba-solver/issues/480

Reply:
Done :)

Reply:
This will be fixed in conda-libmamba-solver 24.7.0, so we can probably require a more recent conda-standalone instead.